spec/system/school/courses/target_details_editor_spec.rb

```diff
@@ -370,4 +370,61 @@
       expect(target.timeline_events.where(id: submission_for_quiz_target_without_grades.id)).to eq([])
     end
   end
+
+  context 'school admin modifies the target group for a target' do
+    let!(:target_group_l1) { create :target_group, level: level_1 }
+    let!(:target_group_l2_1) { create :target_group, level: level_2 }
+    let!(:target_group_l2_2) { create :target_group, level: level_2 }
+    let!(:target_group_archived) { create :target_group, :archived, level: level_2 }
+    let!(:target_l2_1) { create :target, target_group: target_group_l2_1, sort_index: 1 }
+    let!(:target_l2_2) { create :target, target_group: target_group_l2_1, prerequisite_targets: [target_l2_1] }
+    let!(:target_l2_3) { create :target, target_group: target_group_l1, sort_index: 1, prerequisite_targets: [target_l2_2] }
+
+    scenario 'when the new target group is of the same level' do
```

Comment:
This reads like a context, not a scenario. This should probably be: `author moves a target to another group in the same level`.

The next scenario has the same issue.